quesma/quesma/schema_transformer.go

```diff
@@ -386,6 +386,9 @@ func (s *SchemaCheckPass) applyFullTextField(indexSchema schema.Schema, query *m
 			if col.ColumnName == model.FullTextFieldNamePlaceHolder {
 
 				if len(fullTextFields) == 0 {
+					if (e.Op == "LIKE" || e.Op == "ILIKE") && model.AsString(e.Right) == "'%'" {
```

Comment:
Maybe to avoid some subtle bugs?
```suggestion
					if (strings.ToUpper(e.Op) == "LIKE" || strings.ToUpper(e.Op) == "ILIKE") && model.AsString(e.Right) == "'%'" {
```